upup/models/cloudup/resources/addons/networking.projectcalico.org/pre-k8s-1.6.yaml.template

```diff
@@ -69,7 +69,7 @@ spec:
         # container programs network policy and routes on each
         # host.
         - name: calico-node
-          image: quay.io/calico/node:v2.4.0
+          image: quay.io/calico/node:v2.6.2
```

Comment:
If you've managed to test this and show that it works, then I'm OK with it, but note that we haven't extensively tested Calico v2.6.0 on Kubernetes < v1.6, so it's sort of uncharted territory.

I'm not aware of a reason it won't work though.

Reply:
Should we upgrade then?  I think a CVE was fixed in Calico?

Reply:
I think we should stick with the most tested configuration, I assume a CVE would have been fixed in 2.4.x?

Reply:
We need to go to the new policy controller for kops and weave as well.  The old policy controller does not work :(

Reply:
smoke tested 1.5.8 k8s with go-guest book ... worked like a champ.  Will upgrade and let Calico support or tell the users to upgrade clusters :)

Reply:
@chrislovecnm sounds good to me.